cutten-web/src/index.tsx

```diff
@@ -1,7 +1,19 @@
 import * as React from "react";
 import { render } from "react-dom";
+import ApolloClient from "apollo-boost";
+import { ApolloProvider } from "react-apollo";
+
 import "./index.css";
 
 import { App } from "./App";
 
-render(<App />, document.getElementById("root"));
+const client = new ApolloClient({
+    uri: "http://localhost:8000/graphql"
```

Comment:
at some point we will want to drive this off of an environment variable but this is fine for now .